src/DetailsView/reports/components/instance-list-group-header.tsx

```diff
@@ -46,8 +47,10 @@ export const InstanceListGroupHeader = NamedSFC<InstanceListGroupHeaderProps>('I
         return <span className="rule-details-description">{props.ruleResult.description}</span>;
     };
 
+    const ariaLevel = props['aria-level'] || undefined;
+
     return (
-        <div role="heading">
+        <div role="heading" aria-level={ariaLevel}>
```

Comment:
These two attr should come together. If the div have role=heading but no aria level it will fail a11y checks. But this can be taken care of in my PR.

Reply:
Awesome... I didn't want to touch the already existing `role="heading"` part as I'm not fully aware where else we are using this component.